apps/dapp-console-api/src/models/transactions.ts

```diff
@@ -78,6 +80,7 @@ export const transactions = pgTable(
       fromAddressIdx: index().on(table.fromAddress),
       toAddressIdx: index().on(table.toAddress),
       contractAddressIdx: index().on(table.contractAddress),
+      blockTimestampIdx: index().on(table.blockTimestamp),
```

Comment:
curious what's a query pattern where this this index is necessary?

Reply:
might be better to index by blockNumber if the query pattern is look for all transactions from a specific block

Reply:
I added this because the current figma designs have the ui sorting transactions by timestamp and also if we want to paginate these results we'll need a query pattern for querying transactions within a certain time range.

Reply:
is the design for sorting transactions for a single entity?

if the query is sort all transactions **for a given entity** by timestamp we prob want to add a multicolumn index on
(entityId, blockTimestamp). 

having them as two separate indices will not be as helpful with the sorting unless we're sorting the entire table by createdAt